network/p2p/scoring/app_score_test.go

```diff
@@ -111,37 +114,20 @@ func TestFullGossipSubConnectivity(t *testing.T) {
 // the network of honest nodes.
 func TestFullGossipSubConnectivityAmongHonestNodesWithMaliciousMajority(t *testing.T) {
 	// Note: if this test is ever flaky, this means a bug in our scoring system. Please escalate to the team instead of skipping.
-	total := 10
```

Comment:
If we're going to remove the mini load test of 10 runs, then why not just make `testGossipConnectivityUnderNetworkPartition` a full test and fail the test there with the message that `// Note: if this test is ever flaky, this means a bug in our scoring system. Please escalate to the team instead of skipping.`

Is there a reason to keep the `TestFullGossipSubConnectivityAmongHonestNodesWithMaliciousMajority` wrapper if it's just calling `testGossipConnectivityUnderNetworkPartition` once now?